network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/profile_directory.html

```diff
@@ -1,11 +1,19 @@
 {% load static %}
 
 <div class="profile-directory">
-  <div class="profile-filters">
+  <div class="fellowships-directory-filter px-2 mb-5">
     {% for label in filters %}
-    <button class="filter-button">{{ label }}</button>
+    <div class="filter-option">
+      <input type="radio" name="program_year" id="fellow-list-program_year-{{label}}" value="{{label}}" {% if forloop.first %}checked{% endif %}>
+      <label for="fellow-list-program_year-{{label}}">{{label}}</label>
+    </div>
     {% endfor %}
   </div>
 
   {% include "./profile_blocks.html" %}
+
+  {% if filters|length > 0 %}
```

Comment:
Just making sure I understand this: this check is technically unnecessary, since "filters" is a required field on the backend of the component, correct? (If that's the case, I still understand why it's here, I just want to make sure I'm reading this right)

Reply:
correct